### ISSUE TITLE
DigitalOcean Objectfs Error

### ISSUE THREAD

**Issue by akahky** [catalyst/moodle-tool_objectfs#408]
Hello All, 
    I upgraded to moodle 3.10.2 and I am facing a strange issue with objectfs.
the following error is displayed when trying to access the administration module:
**Exception - Class '\tool_objectfs\do_file_system' not found
More information about this error**

Would you please help me to solve this issue?

**Comment by guillefd** [comment]
Have you copied the plugin folder after upgrading?
Have you upgraded the plugin to latest?

**Comment by akahky** [comment]
Yes, I tried both approaches and both did not work, in addition I noticed the old plugin used the name "do_file_system" while the new one is using "digitalocean_file_system". could this be the reason of that issue?
